IopHomeNode.proto3

```diff
@@ -109,10 +117,24 @@ enum Status {
   ERROR_NOT_FOUND = 53;                            // A referenced object was not found.
   ERROR_INVALID_VALUE = 54;                        // One of the values provided is invalid.
   ERROR_ALREADY_EXISTS = 55;                       // An object with the same identifier already exists.
+  ERROR_NOT_AVAILABLE = 56;                        // A request object is not available.
+  ERROR_REJECTED = 57;                             // A third party rejected the requested action.
+  ERROR_UNINITIALIZED = 58;                        // The requested object can not be obtained, queried or used as it is not initialized.
 }
 
 
 // This is complete message as it goes over the wire. It includes the 32-bit header with message length.
+//
+// There is a 1 byte prefix because of Google Protobuf protocol, so the actual binary data for a message
+// with body size 4660 bytes would be:
+//
+//
+//   0D 34 12 00 00
+//
+// where 0x0D means type 'fixed32' (which value is 5) + index of 'header', which is 1 << 3,
+// this is follwed be the 4 byte sequence 34 12 00 00, which is little endian of 0x00001234, which is 4660 decimal.
+// Thus the total number of bytes sent over the wire for this message would be 4665 bytes.
+//
 message MessageWithHeader {
```

Comment:
You are right that it looks unusual, but it is just a comment that can be actually useful for devs using some specific Protobuf implementations (like myself) that can't be used easily for reading whole message including the header from the stream. This is also to stress out the exact format of the header, which was slightly different in previous implementation. Feel free to just ignore this comment and leave it for those devs that may benefit from it.